org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/dashboard/LoginOpenSequenceDialog.java

```diff
@@ -0,0 +1,666 @@
+
+		LayoutUtils.addSclass("btn-small", bDown);
+		LayoutUtils.addSclass("btn-sorttab small-img-btn", bDown);
+		bDown.addEventListener(Events.ON_CLICK, actionListener);
+
+		//
+		ListHead listHead = new ListHead();
+		listHead.setParent(yesList);
+
+		ListHeader listHeader = new ListHeader();
+		listHeader.appendChild(yesLabel);
+
+		Hlayout yesButtonLayout = new Hlayout();
+		yesButtonLayout.appendChild(bUp);
+		yesButtonLayout.appendChild(bDown);
+		yesButtonLayout.setStyle("display: inline-block; float: right;");
+		listHeader.appendChild(yesButtonLayout);
+		listHeader.setParent(listHead);
+
+		//
+		listHead = new ListHead();
+		listHead.setParent(noList);
+
+		listHeader = new ListHeader();
+		listHeader.appendChild(noLabel);
+		listHeader.setSort("auto");
+		listHeader.setSortDirection("ascending");
+		// http://books.zkoss.org/wiki/ZK%20Configuration%20Reference/zk.xml/The%20Library%20Properties/org.zkoss.zul.listbox.autoSort
+		noList.setAttribute("org.zkoss.zul.listbox.autoSort", "true");
+		Hlayout noButtonLayout = new Hlayout();
+		noButtonLayout.appendChild(bRemove);
+		noButtonLayout.appendChild(bAdd);
+		noButtonLayout.setStyle("display: inline-block; float: right;");
+		listHeader.appendChild(noButtonLayout);
+		listHeader.setParent(listHead);
+
+		// Confirm Panel listener
+		EventListener<Event> onClickListener = new EventListener<Event>() {
+			@Override
+			public void onEvent(Event event) throws Exception
+			{
+				if (event.getTarget().equals(confirmPanel.getButton(ConfirmPanel.A_OK)))
+				{
+					saveData();
+				}
+				else if (event.getTarget().equals(confirmPanel.getButton(ConfirmPanel.A_CANCEL)))
+				{
+					detach();
+				}
+				else if (event.getTarget().equals(confirmPanel.getButton(ConfirmPanel.A_RESET)))
+				{
+					loadData();
+				}
+			}
+		};
+
+		confirmPanel.addActionListener(onClickListener);
+	} // init
+
+	/**
+	 * Load data into model
+	 */
+	public void loadData()
+	{
+		autoOpenSeqs.clear();
+		yesModel.removeAllElements();
+		noModel.removeAllElements();
+
+		int favTreeID = MTreeFavorite.getFavoriteTreeID(m_AD_User_ID);
+		MRole role = MRole.get(Env.getCtx(), Env.getAD_Role_ID(Env.getCtx()));
+
+		// get login open seq no nodes
+		Query query = new Query(Env.getCtx(), MTreeFavoriteNode.Table_Name, "AD_Tree_Favorite_ID=? AND AD_Menu_ID>0 AND LoginOpenSeqNo>=0", null);
+		query.setOrderBy(MTreeFavoriteNode.COLUMNNAME_LoginOpenSeqNo);
+		query.setOnlyActiveRecords(true);
+		query.setParameters(new Object[] { favTreeID });
+
+		for (int favNodeID : query.getIDs())
+		{
+			autoOpenSeqs.add(favNodeID);
+		}
+
+		// get all menu nodes
+		query = new Query(Env.getCtx(), MTreeFavoriteNode.Table_Name, "AD_Tree_Favorite_ID=? AND AD_Menu_ID>0 ", null);
+		query.setOrderBy(MTreeFavoriteNode.COLUMNNAME_LoginOpenSeqNo);
+		query.setOnlyActiveRecords(true);
+		query.setParameters(new Object[] { favTreeID });
+		List<MTreeFavoriteNode> lsFavNode = query.list();
```

Comment:
During tests with SuperUser in a non-GardenWorld client, this line threw error:
Cross tenant PO reading request detected from session 1000059 for table AD_Tree_Favorite_Node Record_ID=200021